learn/toc.yml

```diff
@@ -38,3 +38,7 @@
       href: chapter2/5-light-casters.md
     - name: 2.6. Multiple Lights
       href: chapter2/6-multiple-lights.md
+- name: "Appendix: Misc. OpenGL"
+  items:
+    - name: Debug Callback, Messages, and Object Labels
```

Comment:
This should be `Debug Callback` for this article and we can add more of them in the future. Better to keep the articles focused.